### PR TITLE
Adds interactive examples to pseudo class pages

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Adds the interactive examples which were made available a month ago by @NiedziolkaMichal https://github.com/mdn/interactive-examples/pull/2130.

### CODE REVIEW

File: files/en-us/web/css/_colon_first-child/index.md [hunk 1]
```diff
@@ -14,16 +14,10 @@ browser-compat: css.selectors.first-child
...
 
-```css
-/* Selects any <p> that is the first element
-   among its siblings */
-p:first-child {
-  color: lime;
-}
-```
-
 > **Note:** As originally defined, the selected element had to have a parent. Beginning with Selectors Level 4, this is no longer required.
```
Comment (wbamberg) [inline files/en-us/web/css/_colon_first-child/index.md:17]: If we still need them, I would prefer these notes to go after the example, not before.

Comment (teoli2003) [reply files/en-us/web/css/_colon_first-child/index.md:17]: I'm pretty sure this note is no more relevant.

Comment (wbamberg) [reply files/en-us/web/css/_colon_first-child/index.md:17]: It's also not very actionable. As a developer I'm not particularly interested in Selectors Level 4, I'm interested in browser support.

Comment (OnkarRuikar) [reply files/en-us/web/css/_colon_first-child/index.md]: I've removed the notes.

**Review by wbamberg — APPROVED** [review]
👍 thank you @OnkarRuikar and @NiedziolkaMichal , looks great!

**Pull request merged**